### ISSUE TITLE
Moto g42 crdroid 

### ISSUE THREAD

**Issue by DAB006** [HemanthJabalpuri/twrp_motorola_rhode#10]
Can I install crdroid without flashing twrp
Crdroid have a recovery file

**Comment by HemanthJabalpuri** [comment]
> Can I install crdroid without flashing twrp Crdroid have a recovery file

Yes, you can. Temporary booting with `fastboot boot TWRP.img` will not install TWRP to the device.

**Comment by DAB006** [comment]
Crdroid have recovery too right 
Will that work?

**Comment by HemanthJabalpuri** [comment]
You have to ask that in crDroid support.
